module.txt

```diff
@@ -50,6 +50,11 @@
             "id": "LightAndShadowResources",
             "minVersion": "1.1.0"
         },
+        {
+            "id": "Notifications",
+            "minVersion": "0.1.0",
+            "maxVersion": "0.3.0"
```

Comment:
What's the reason to have a version range here?

Reply:
nothing in particular, was just following the ranges specified when being used in the other modules (Eventual Skills and Ingame Help)

Reply:
Okay, then I guess I have to ask myself why I put them in there that way :thinking: :sweat_smile: 

I guess with the leading `0.` each minor version upgrade is incompatible, and since I knew that 0.1.0 and 0.2.0 are still backwards compatible I put in this range ...